app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Post should belong to author not user `belongs_to :author, class_name: 'User'`